index.bs

```diff
@@ -5338,18 +5348,24 @@ The <dfn id="dfn-typed-array-type" export>typed array types</dfn> are
 {{Uint16Array}},
 {{Uint32Array}},
 {{Uint8ClampedArray}},
-{{Float32Array}} and
-{{Float64Array}}.
+{{Float32Array}},
+{{Float64Array}},
+any [=nullable types=] whose [=nullable types/inner type=] is a [=typed array type=],
```

Comment:
But here we couldn't simplify overload resolution step 12.7 because for the annotated/nullable types the name would not match...